basiclti/basiclti-blis/src/java/org/sakaiproject/lti13/LTI13Servlet.java

```diff
@@ -1283,8 +1281,9 @@ protected void handleNamesAndRoles(String signed_placement, HttpServletRequest r
 				JSONArray roles = new JSONArray();
 
 				// If there is a role mapping, it has precedence over site.update
-				if ( roleMap.containsKey(role.getId()) ) {
-					roles.add(roleMap.get(role.getId()));
+				String actual_role = role.getId();
+				if ( roleMap.containsKey(actual_role) ) {
```

Comment:
Why do you mix camel case and underscores in your variable names? Just out of interest? Do they have different meanings in the LTI world?

Reply:
So -- there is a method to my madness - not necessarily a brilliant idea, but.  I tend to use underscore when I am preparing variables that will end up in JSON.  This may or may not be the case for actual_role :)